### ISSUE TITLE
Saving not working in publishing pop-up

### ISSUE THREAD

**Issue by cessda-bitbucket-importer** [cessda/cessda.cvs.two#322]
**Original report on BitBucket by Taina Jääskeläinen.**

----------------------------------------

Making a new SL version: when 1\) clicking on ‘Save’ in publishing pop-up and 2\) reopening the publishing pop-up later

* Text entered or copy pasted into _Version changes_ or V_ersion notes_ elements has disappeared
* Change logs information has disappeared and the log is empty
* Compare table is still correct
* Version number manual changes have been retained, so correct

If the user clicks straight on ‘Publish’ button without saving, text entered into _Version changes_ or _Version notes_ is saved and visible in the published version. Even those changes that disappeared when clicking on ‘Save’ and not re-entered are there.  So the system saves the changes when the user clicks on ‘Save’ but does not show them when the user re-opens the pop-up.

So in short: ‘Publish’ functionality works well but ‘Save’ functionality does not show saved text in the text fields in the next pop-up and empties the change log.

‌
